articles/tales-from-fleet-security-securing-1password.md

```diff
@@ -0,0 +1,81 @@
+# Tales from Fleet security: securing 1Password
+
+![Securing 1Password](http://fleetdm.com/images/articles/securing-1password-1600x900@2x)
+
+No matter how much we wish SaaS tools had support for Single Sign-On (SSO), there are still so many
+websites and other resources that require individual passwords. Running a company without providing
+employees with a password manager is downright dangerous.
+
+At Fleet, we use 1Password. While configuring 1Password is relatively straightforward, here are a few things we do that can help you secure your 1P instance.
```

Comment:
Is 1P a recognized abbreviation? It made me stop and think for a moment while reading. I suggest sticking with 1Password for the sake of clarity.